server/tserver/src/main/java/org/apache/accumulo/tserver/ScanServer.java

```diff
@@ -601,7 +602,13 @@ private Map<KeyExtent,TabletMetadata> reserveFilesInner(Collection<KeyExtent> ex
       }
 
       if (!filesToReserve.isEmpty()) {
-        getContext().getAmple().putScanServerFileReferences(refs);
+        long reservationWriteStart = System.nanoTime();
+        try {
+          getContext().getAmple().putScanServerFileReferences(refs);
```

Comment:
Micrometer Timer meters have a `record(Runnable)` method that could possibly be used here by passing it a lambda that calls `putScanServerFileReferences()`.  Not sure if that is shorter or better, this is stylistic comment feel free to ignore it.

Reply:
I think it does make things shorter and cleaner. I'll make this change.